operations_on_datastructures/array_left_rotation_efficient.cpp

```diff
@@ -0,0 +1,34 @@
+#include<iostream>
+
```

Comment:
Please make comments before name of variable and function and give description. What is the use of this variable and function. It will be easy for people who see your code.

Reply:
Sure, Thanks for your input.

Reply:
Done.